src/read.rs

```diff
@@ -165,81 +166,81 @@ impl Description {
     pub fn sample_rate(&self) -> u32 {
         self.sample_rate
     }
-
 }
 
-
 impl BufFileReader {
-
     /// Attempts to open an audio `Reader` from the file at the specified `Path`.
     ///
     /// This function is a convenience wrapper around the `Reader::new` function.
     ///
     /// This function pays no attention to the `file_path`'s extension and instead attempts to read
     /// a supported `Format` via the file header.
     pub fn open<P>(file_path: P) -> Result<Self, ReadError>
-        where P: AsRef<std::path::Path>,
+    where
+        P: AsRef<std::path::Path>,
     {
         let path = file_path.as_ref();
-        let file = try!(std::fs::File::open(path));
+        let file = r#try!(std::fs::File::open(path));
         let reader = std::io::BufReader::new(file);
         Reader::new(reader)
     }
-
 }
 
 impl<R> Reader<R>
-    where R: std::io::Read + std::io::Seek,
+where
+    R: std::io::Read + std::io::Seek,
 {
-
     /// Attempts to read the format of the audio read by the given `reader` and returns the associated
     /// `Reader` variant.
     ///
     /// The format is determined by attempting to construct each specific format reader until one
     /// is successful.
     pub fn new(mut reader: R) -> Result<Self, ReadError> {
-
-        #[cfg(feature="wav")]
+        #[cfg(feature = "wav")]
         {
             let is_wav = match hound::WavReader::new(&mut reader) {
                 Err(hound::Error::FormatError(_)) => false,
                 Err(err) => return Err(err.into()),
                 Ok(_) => true,
             };
-            try!(reader.seek(std::io::SeekFrom::Start(0)));
+            r#try!(reader.seek(std::io::SeekFrom::Start(0)));
```

Comment:
And another - it looks like there's quite a few of these - it might be worth searching for al instances of `try!` and replacing them with `?` use. In the meantime I'll see if I can find the purpose of this `r#` syntax, I'm obviously missing something!